frontend/src/staging/StagingBrowserNav.vue

```diff
@@ -66,12 +70,17 @@ export default class StagingBrowserNav extends Vue {
     @Prop({ default: false }) isFileSelected!: boolean;
 
     get workingDirectoryArray(): string[] {
```

Comment:
braucht man wirklich diese funktion ? die macht ja nichts anderes als ein split. Und wenn der string leer ist sollte vielleicht eine exception werfen. dann könnte man besser ein zukünftiger Fehler entdecken.

Reply:
Ist es denn ein Fehler, dass this.workingDirectory leer ist? Kann das nicht bei der Initialisierung der Fall sein? Bevor man einen Ordner ausgewählt hat.

Reply:
@preiaen: Es ist kein Fehler. Das wäre einfach der Fall wenn man die Staging-Seite gerade geladen hat. 

Das hatte ich eingefügt weil ein Split an der Wurzel des Verzeichnisbaumes  `''.split('/') == ['']`. Wenn man allerdings einen Pfad mit Slash hat ergibt das z.B.` 'Pfad/Test/Hier'.split('/') == ['Pfad', 'Test', 'Hier']` anstatt `['', 'Pfad', 'Test', 'Hier']`. Das macht vielleicht Sinn im Kontext der Split Funktion, für unseren Code liest sich das dann aber so als wäre "Pfad" die Wurzel und es kommt zu Fehlern im Interface.